pkg/metrics/metrics.go

```diff
@@ -33,8 +33,32 @@ var (
 		Name: "mapi_mao_collector_up",
 		Help: "Machine API Operator metrics are being collected and reported successfully",
 	}, []string{"kind"})
+
+	// FailedMachineSetProvisionCount calculates the number of failures for a specific machine
```

Comment:
Should this say MachineSet?
```suggestion
	// FailedMachineSetProvisionCount calculates the number of failures for a specific MachineSet
```

Reply:
Removed those, as these metrics will only add additional clutter on already present failure reporting in metrics for machine instances.